src/styles/blocks/product_item.scss

```
@@ -0,0 +1,91 @@
+.product-item {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Border-radius used twice in the code: for photo and button, so you should create variable.
```suggestion
  border-radius: $border-radius;
```